gtsfm/data_association/point3d_initializer.py

```diff
@@ -192,13 +205,46 @@ def triangulate(self, track_2d: SfmTrack2d) -> Tuple[Optional[SfmTrack], Optiona
 
         # all the measurements should have error < threshold
         if not np.all(reproj_errors < self.reproj_error_thresh):
-            return None, avg_track_reproj_error, is_cheirality_failure
+            return None, avg_track_reproj_error, TriangulationResult.HIGH_REPROJECTION_ERROR
 
         track_3d = SfmTrack(triangulated_pt)
         for i, uv in inlier_track.measurements:
             track_3d.add_measurement(i, uv)
 
-        return track_3d, avg_track_reproj_error, is_cheirality_failure
+        return track_3d, avg_track_reproj_error, TriangulationResult.SUCCESS
+
+    def is_small_triangulation_angle(self, point_3d: np.ndarray, camera_indices: List[int]) -> bool:
+        """Check if all pairs of cameras have a small triangulation angle.
+
+        References:
+        - https://github.com/colmap/colmap/blob/513fef3e39f99b50279bf5aee1c5b597cba53c3b/src/base/reconstruction.cc#L616
```

Comment:
maybe we can add a link to the implementation of the COLMAP fn here, also: https://github.com/colmap/colmap/blob/dev/src/base/reconstruction.cc#L1412

Reply:
Like `"Based off of the implementation here: https://github.com/colmap/colmap/blob/dev/src/base/reconstruction.cc#L1412"`

Reply:
Sure